src/main/java/frc/robot/vision/multivisionsources/MultiVisionSources.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.vision.multivisionsources;
+
+import frc.constants.VisionConstants;
+import frc.robot.vision.data.VisionData;
+import frc.robot.vision.sources.VisionSource;
+import org.littletonrobotics.junction.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+
+public class MultiVisionSources<ReturnType extends VisionData> {
+
+	private final String logPath;
```

Comment:
```suggestion
	protected final String logPath;
```

same reason as `visionSources`